### DISCUSSION TITLE
UDP port support

### DISCUSSION THREAD

**Discussion by tristan-f-r** [community/community#44825] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Developing applications with WebRTC in codespaces is a pain without proper UDP port access. The only other suitable alternatives is to host our own STUN and TURN servers. This also makes it hard to develop other applications (e.g. games that use UDP as a means of communicating large visual data).

**Comment by Outfit8TSB** [comment]
yeah i need to run geyser in codespace
